src/pydicom/filereader.py

```diff
@@ -602,7 +639,7 @@ def _read_command_set_elements(fp: BinaryIO) -> Dataset:
 
     def _not_group_0000(tag: BaseTag, vr: str | None, length: int) -> bool:
         """Return True if the tag is not in group 0x0000, False otherwise."""
-        return tag.group != 0
+        return tag >> 16 != 0
```

Comment:
I wonder if there is a nicer way for this kind of inlining optimization (not for this one, this is readable enough). Would dataclasses be better?